Sources/ExyteChat/ChatView/Recording/RecordWaveform.swift

```diff
@@ -53,20 +56,29 @@ struct RecordWaveformWithButtons: View {
 }
 
 struct RecordWaveformPlaying: View {
-
+    
```

Comment:
Please preserve the selected code style, namely:
- do not add extra spaces (this is set up in xcode)
- do not add extra \n like here:
```
var body: some View {

        GeometryReader { g in
```
- after // there should be a space
- after // there is a small letter
- viewPadding:CGFloat - there should be a space here: "viewPadding: CGFloat"
- there should be a \n between these lines:
```
@StateObject var recordPlayer = RecordingPlayer()
//160 is screen left-padding/right-padding and playButton's width.
```